app/Actions/Place/GetPlaceItem/GetPlaceItemPresenter.php

```diff
@@ -61,7 +61,7 @@ public function present(GetPlaceItemResponse $placeResponse): array
         $result['features'] = $this->featurePresenter->presentCollection($place->features);
         $result['localization'] = $this->localizationPresenter->presentCollection($place->localization);
         $result['category'] = $this->categoryPresenter->present($place->category);
-        $result['category']['tags'] = $this->tagsPresenter->presentCollection($place->category->tags);
+        $result['category']['tags'] = $this->tagsPresenter->presentCollection($place->tags);
```

Comment:
[place][tags]

Reply:
Corrected there and deeper also (in UserLists pages).